### PR TITLE
Podspec update

### PR DESCRIPTION
**Pull request by khawkins**
I made some changes, and the podspec lints out okay.  I'll add a couple of notes.

### CODE REVIEW

File: SalesforceMobileSDK-iOS.podspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "7.0"
 
   s.source       = { :git => "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git",
-                     :tag => "pod_v#{s.version}",
+                     :tag => "v#{s.version}",
```
Comment (khawkins) [inline SalesforceMobileSDK-iOS.podspec]: I changed this back, in anticipation that we'll be re-aligned with the release version tag when this moves forward.

File: libs/SalesforceNetwork/SalesforceNetwork.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -74,6 +74,10 @@
 		6FD16C7D1AA4EF4E001628F3 /* CSFAvailability.h in Headers */ = {isa = PBXBuildFile; fileRef = 827DD4751A7C006F00519F03 /* CSFAvailability.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		6FD16C951AA4EF88001628F3 /* SalesforceSDKCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6FD16C901AA4EF73001628F3 /* SalesforceSDKCore.framework */; };
 		6FD16CAB1AA4EFB5001628F3 /* SalesforceOAuth.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6FD16CA61AA4EFA8001628F3 /* SalesforceOAuth.framework */; };
+		820053F81ACDBE7200C96578 /* NSDate+ActionValue.h in Headers */ = {isa = PBXBuildFile; fileRef = 820053F61ACDBE7200C96578 /* NSDate+ActionValue.h */; };
```
Comment (khawkins) [inline libs/SalesforceNetwork/SalesforceNetwork.xcodeproj/project.pbxproj]: This guy somehow wasn't in the project, but CocoaPods was grabbing it, then complaining about a bug (which you'll see the fix in the file below).

**Comment by bhariharan** [comment]
:+1:

**Comment by bhariharan** [comment]
Should I merge this in?

**Comment by khawkins** [comment]
Yeah, it's good to go.

**Pull request merged**